### ISSUE TITLE
Can I have a newest version of swiftmailer (^2.0.0 or ~2.1.0 instead of ~2.0.0)

### ISSUE THREAD

**Issue by bscheshirwork** [yiisoft/yii2-app-advanced#296]
### What steps will reproduce the problem?
I have a problem with composer update now

### What's expected?
successful update

### What do you get instead?
```sh
  [Composer\DependencyResolver\SolverProblemsException]                                                                                                             
    Problem 1                                                                                                                                                       
      - The requested package yiisoft/yii2-swiftmailer ~2.0.0 exists as yiisoft/yii2-swiftmailer[2.1.x-dev, dev-master] but these are rejected by your constraint.  
```

### Additional info

| Q                | A
| ---------------- | ---
| Yii vesion       | yiisoft/yii2                            dev-master 6436631 Yii PHP Framework Version 2
||yiisoft/yii2-swiftmailer                dev-master a752474 The SwiftMailer integration for the Yii framework
| PHP version      | 7.1.8
| Operating system | alpine

**Comment by cebe** [comment]
you can of course update it in your local copy. We can make this change for the template in 2.1.

**Comment by bscheshirwork** [comment]
So... I can propose dependency 
```json
    "require": {
        "php": ">=7.0.0"
    },
```
in branch of #297?

**Comment by cebe** [comment]
fixed by #297
